gub/3/new.py

```diff
@@ -1,4 +1,4 @@
-# https://docs.python.org/3.0/whatsnew/3.0.html
+# http://docs.python.org/3.0/whatsnew/3.0.html
```

Comment:
Ok, in that case could you rebase that commit into the previous one?

```
git rebase -i master
```
is what I do.